src/components/tooltip/tooltip.js

```diff
@@ -66,11 +59,11 @@ export const BTooltip = /*#__PURE__*/ Vue.extend({
       default: () => getComponentConfig(NAME, 'boundary')
     },
     boundaryPadding: {
-      type: [Number, String],
+      type: Number,
       default: () => getComponentConfig(NAME, 'boundaryPadding')
     },
     offset: {
-      type: [Number, String],
+      type: Number,
```

Comment:
We should allow Numbers passed as strings

Reply:
Shouldn't it be possible to pass both `skidding` and `distance`? This would allow positing both vertical and horizontal.

Documentation: https://popper.js.org/docs/v2/modifiers/offset/#options